archive/f/f-sharp/README.md

```diff
@@ -5,6 +5,7 @@ Welcome to Sample Programs in F#!
 ## Sample Programs
 
 - [Hello World in F#](https://github.com/jrg94/sample-programs/issues/104)
+- [List search in F#](https://github.com/TheRenegadeCoder/sample-programs/blob/master/archive/f/f-sharp/search.fs)
```

Comment:
Could you make a new article issue like #33 and link it here instead of the solution itself.